assets/src/edit-story/output/utils/styles.js

```diff
@@ -52,7 +52,8 @@ function CustomStyles() {
 
               @media (max-aspect-ratio: 9 / 16)  {
                 @media (min-aspect-ratio: 1 / 2) {
```

Comment:
How did you come up with the values `9/16` and `1/2`?

Reply:
Got the lower bound (1 / 2) ratio by testing on UI. (Anything that is bellow 1 / 2 looks good at it is for now)

And got the upper bound (9 / 16) ratio by following the styling on the fullbleed area. (anything that is above 9 / 16 will be handled nicely by the default style)

I will wrap the ratio text into variables and add some comments.

Reply:
Can you add screenshots for the upper/lower bound scenarios?